sdk/eventhub/azure-eventhub/tests/pyamqp_tests/unittest/test_exceptions_pyamqp.py

```diff
@@ -12,7 +12,7 @@ def test_client_creation_exceptions():
         sender = SendClient(
             "fake.host.com",
         )
-        assert sender._hostname == "fake.host.com"
+        assert sender._remote_address == "fake.host.com"
```

Comment:
(fix) like the tests below -- target I think - or remove assert? We might not need this test anymore - I will play around with it